src/rantanplan/core.py

```diff
@@ -201,7 +222,7 @@ def get_stresses(phonological_groups):
     # Get position for the last syllable of the penultimate word
     if last_word_syllables.count(True) > 1:
         penultimate_word = -(
-            [i for i, n in enumerate(last_word_syllables[::-1]) if n][1] + 1)
+                [i for i, n in enumerate(last_word_syllables[::-1]) if n][1] + 1)
```

Comment:
Why the double indent?

Reply:
Oops sorry I tried reverting all indentation line>80 changes and this was left out